docker-compose.ci.yml

```diff
@@ -0,0 +1,3 @@
+version: '3'
+
+services: {}
```

Comment:
Required by `dockerComposeProjectTestsParallel`. Should be a standard so no need to make it optional.